src/Controls/src/Core/Platform/GestureManager/GestureManager.iOS.cs

```diff
@@ -230,6 +231,37 @@ public void Dispose()
 				return tapGestureRecognizer;
 			}
 
+			var pointerGestureRecognizer = recognizer as PointerGestureRecognizer;
+			if (pointerGestureRecognizer != null)
+			{
+				var uiRecognizer = CreatePointerRecognizer(r =>
+				{
+					if (weakRecognizer.Target is PointerGestureRecognizer pointerGestureRecognizer &&
+						weakEventTracker.Target is GestureManager eventTracker &&
+						eventTracker._handler?.VirtualView is View view &&
+						UIApplication.SharedApplication.GetKeyWindow() is UIWindow window)
```

Comment:
```suggestion
						eventTracker._handler?.MauiContext?.GetPlatformWindow() is UIWindow window)
```

Reply:
this will retrieve the window from the element instead of globally which is better for multi window